src/components/chart/examples/chart-items-creative-styling.ts

```diff
@@ -26,26 +26,23 @@ export const stackedBarChartItems: ChartItem[] = [
     },
 ];
 
-export const ganttChartItems: ChartItem[] = [
+export const ganttChartItems: Array<ChartItem<[number, number]>> = [
     {
         text: 'Week 1',
         prefix: '%',
-        startValue: -7,
-        value: 10,
+        value: [-7, 10],
```

Comment:
👍 
there are places in the docs where I have mentioned `minValue`, which we should also change